src/main/java/domain/LottoMachine.java

```diff
@@ -0,0 +1,31 @@
+package domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class LottoMachine {
+
+    private final static int MIN_LOTTO_NUMBER = 1;
+    private final static int MAX_LOTTO_NUMBER = 45;
+
+    private LottoMachine() {}
+
```

Comment:
```suggestion
public class LottoMachine {

    private static final int MIN_LOTTO_NUMBER = 1;
    private static final int MAX_LOTTO_NUMBER = 45;

    private LottoMachine() {}

```
생성시 혹은 ```generate()```, ```LottoNumberGenerator```을 파라미터로 넘겨주는 것은 어떨까요? 
그리고 private static final 순서는 컨벤션입니다! ```prsf``` 로 자동완성도 가능해요!